### DISCUSSION TITLE
How to upload images to Wiki?

### DISCUSSION THREAD

**Discussion by hius07** [koreader/koreader#7742] [Q&A]
In the web browser, to https://github.com/koreader/koreader/wiki/screenshots?

**Comment by Frenzie** [comment]
I don't know if you can; perhaps you can just drag into the box? The wiki is a full git repo, so you can use git.

**Reply by hius07** [reply]
Can we have in wiki the same bottom line (Attach files...) as in these comments?

**Reply by hius07** [reply]
I'm talking about the browsers only.

**Reply by Frenzie** [reply]
You'd have to ask GitHub. Not our choice. ;-)

**Reply by hius07** [reply]
> perhaps you can just drag into the box?

Have tried. No, wiki editing window doesn't allow dropping, the browser opens the dragged image in a new tab.

**Comment by Galunid** [comment]
One way I know is to upload them somewhere (e.g. imgur) and use standard markdown `![](direct_image_url)` to have it displayed. I think you can't directly upload image to github (wiki). When you click 
![image](https://user-images.githubusercontent.com/10298730/119274261-44d0d480-bc0f-11eb-8b09-a67fac81aa6e.png)
prompt that appears asks for image url:
![image](https://user-images.githubusercontent.com/10298730/119274272-5619e100-bc0f-11eb-82ec-d33e184e5029.png)

You can probably do something like uploading image to issue (without posting) and using link that appears in issue box in wiki, but I don't know how well that would work

**Reply by hius07** [reply]
> uploading image to issue

Yes, I've done so in my previous articles, and got images at https://user-images.githubusercontent.com, not at the wiki/screenshots.
Seems that browsers cannot do that, only git can.
